apps/src/templates/manageStudents/MoveStudents.jsx

```diff
@@ -249,13 +244,20 @@ class MoveStudents extends Component {
     });
   };
 
+  isMovableSection = (section) => {
+    if (section.id === this.props.currentSectionId) { return false; }
+    if (section.loginType === SectionLoginType.google_classroom || section.loginType === SectionLoginType.clever) { return false; }
+    return true;
+  };
+
   renderOptions = () => {
-    const {sections, currentSectionId} = this.props;
-    let options = sections.map(section => {
-      if (section.id === currentSectionId) {
-        return null;
-      } else {
+    const {sections} = this.props;
+    let options = Object.keys(sections).map(sectionId => {
```

Comment:
I wonder if we should have a group conversation about whether we want to use the JavaScript semantics for `let/const` (where this should be `const` because we never reassign the variable), or if we want to depend on our own semantic for it (where this should be `let` since we modify the array later).  My personal feeling is to do the former because it gives us the strictest enforcement the language allows, but I'm not sure what anyone else thinks yet.

Reply:
I'm pro-`const` in this case, since `options` is never reassigned. If we use `let` here, I'd be worried it would lead to confusion over what `let` really means. Happy to have a group discussion about it.